chapters/12-project-management.Rmd

```diff
@@ -11,4 +11,17 @@
 
 <!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
 
+1. Create a [project roadmap](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/). 
+- Create a roadmap for your project describing short-term, medium-term, and long-term project goals. This can be a GitHub issue or a file called ROADMAP.md. Example: [STEMMM Role Models roadmap](https://github.com/KirstieJane/STEMMRoleModels/issues/1).
 
+2. Issue tracking and labeling 
+- Create several GitHub issues for your project using your roadmap for inspiration. These can range from simple project improvements (like scanning for typos in the README) to complex feature additions or bugs to be fixed. 
+- Create several issue labels that make sense for your project, i.e. good first bug, discussion, help wanted, documentation, feature, etc. Label your issues with one or more appropriate labels.
+- Add a link to one or more issue categories to CONTRIBUTING.md. Add a link to issues that a new contributor could attempt, i.e. "good first bug".
+- Add links to your issues and/or issue labels to your roadmap.
+
+3. Create GitHub milestones
```

Comment:
I see milestones and projects in GitHub as different ways of doing the same thing. Perhaps we could include a question that asks which of these two options feel better for this project, or ask them to explain how the two approaches could work together?

Reply:
Agreed, especially when it comes to GitHub (I feel GitLab they do different things compared to how GitHub implements them).

Reply:
Good point - I changed the question to be looking at examples and choosing which one they like or suits their project and then implementing that one.